examples/platform/bouffalolab/common/plat/plat.h

```diff
@@ -23,6 +23,10 @@
 
 #define EXT_DISCOVERY_TIMEOUT_SECS 20
 
+#ifndef CONFIG_APP_FABRIC_TABLE_DELEGATE
+#define CONFIG_APP_FABRIC_TABLE_DELEGATE 0
```

Comment:
Shouldn't this be named something like CONFIG_APP_SKIP_ADVERTISING_COMMISSIONABLE_ON_LAST_FABRIC_REMOVAL?

Because it's not really clear from the name or the (nonexistent) documentation what the implications of setting this to 1 or leaving it at the default value of 0 are.

Reply:
Named MACRO to `CONFIG_APP_ADVERTISE_COMMISSIONABLE_ON_LAST_FABRIC_REMOVAL` and set this function disabled by default. Application needs explicitly enable this function if open commission window on last fabric removal is its expected behavior.